tests/unit/state/test_graph_combinatorics.py

```diff
@@ -0,0 +1,36 @@
+# pylint: disable=redefined-outer-name
+
+import pytest
+
+from expertsystem.state.particle import initialize_graph
+from expertsystem.topology import (
+    InteractionNode,
+    SimpleStateTransitionTopologyBuilder,
+    Topology,
+)
+
+
+@pytest.fixture(scope="package")
+def two_body_decay() -> Topology:
```

Comment:
This is actually a 1 to 3 body decay topology. So the name is a bit misleading to me

Reply:
e9715e8
Now I'm thinking whether it wouldn't be better to test a 2-to-3 body decay instead of a 1-to-4 in `test_topology.py::TestTopology` :thinking:

Reply:
5333e4fc

Reply:
ok good